tests/kola/version/rhaos-pkgs-match-openshift

```diff
@@ -9,6 +9,11 @@ set -xeuo pipefail
 
 . $KOLA_EXT_DATA/commonlib.sh
 
-if [[ $(rpm -qa | grep rhaos | grep -v $OPENSHIFT_VERSION) ]]; then
+# source the environment variables to get OPENSHIFT_VERSION
+source /etc/os-release
+
+rpm -qa | grep "rhaos" > rhaos-packages.txt
+
+if grep -qv "${OPENSHIFT_VERSION}" rhaos-packages.txt; then
```

Comment:
I think matching rhaos${OPENSHIFT_VERSION} will be more solid. That will avoid to match the OCP version in the package version. 
For example if OCP version is 4.16 and a package has a version containing 4.16 this would not trigger an error `foobar-4.16.4-0.1.rhaos4.13.el9.x86_64`

```suggestion
if grep -qv "rhaos${OPENSHIFT_VERSION}" rhaos-packages.txt; then
```

Reply:
Also I think dropping the -q (quiet) option might be good so that when the test fail we can see which packages trigger the error in the logs. That makes it easier to investigate.

```suggestion
if grep -v "${OPENSHIFT_VERSION}" rhaos-packages.txt; then
```

Reply:
Thank you - I agree with the above.